utils/mapitest/modules/module_zentyal.c

```diff
@@ -587,3 +588,164 @@ _PUBLIC_ bool mapitest_zentyal_6723(struct mapitest *mt)
 
 	return test_result;
 }
+
+/**
+    \details Test #8174 ModifyRecipients and try to build RecipientRow
+    with no additional properties.
+
+    It is identified as CR-1572 (https://projects.zentyal.com/issues/3681)
```

Comment:
That's a private url, I think this should be https://tracker.zentyal.org/issues/3681